pkg/client/query.go

```diff
@@ -10,41 +10,16 @@ import (
 	"google.golang.org/grpc"
 )
 
-// func (c *Client) QueryAccount(address string) (*at.BaseAccount, error) {
```

Comment:
Does speedway use this?

Reply:
It was commented out so I don't think so.